client/trino-jdbc/src/main/java/io/trino/jdbc/ConnectionProperties.java

```diff
@@ -610,6 +612,15 @@ public Source()
         }
     }
 
+    private static class IgnoreBrokenCatalogs
+            extends AbstractConnectionProperty<Boolean>
+    {
+        public IgnoreBrokenCatalogs()
+        {
+            super("ignoreBrokenCatalogs", Optional.of("false"), NOT_REQUIRED, ALLOWED, BOOLEAN_CONVERTER);
```

Comment:
This doesn't seem to be used.

Reply:
It is actually used to in https://github.com/mdesmet/trino/blob/9ab23053c95e7c969d26bc475ebeeae7ec56a42c/client/trino-jdbc/src/main/java/io/trino/jdbc/ConnectionProperties.java#L91